styles/new/base.css

```diff
@@ -79,3 +79,38 @@ table {
     @apply border-l-0;
   }
 }
+
+h1 {
```

Comment:
Ah nevermind!

Reply:
But still maybe having a desicated typography file idunno

Reply:
It would definitely be better to have it in its file, I'll update it

Reply:
I only moved the styles related to typography, if there is any styling that you think should be added, I can add it